lib/reporters/mini.js

```diff
@@ -77,6 +78,8 @@ MiniReporter.prototype.prefix = function (str) {
 MiniReporter.prototype._test = function (test) {
 	var title = test.title;
 
+	title = cliTruncate(title, process.stdout.columns - 3);
```

Comment:
Should truncate `test.title` and assign the result to `var title`, rather than reassigning `title`.